### PR TITLE
New package: quickDocs-2.1.1

### PR DESCRIPTION
**Pull request by TeddyDD**
basically it's devdocs.io + valadoc in webkitgtk wrapper, useful litte tool

### CODE REVIEW

File: srcpkgs/quickDocs/template [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'quicDocs'
+pkgname=quickDocs
+version=2.1.1
+revision=1
+build_style=meson
+hostmakedepends="libarchive-devel gtk+3-devel devhelp-devel webkit2gtk-devel
+ meson vala pkg-config"
+makedepends=""
+depends=""
```
Comment (maxice8) [inline srcpkgs/quickDocs/template]: remove empty variables

File: srcpkgs/quickDocs/template [hunk 2]
```diff
@@ -0,0 +1,15 @@
+# Template file for 'quicDocs'
+pkgname=quickDocs
+version=2.1.1
+revision=1
+build_style=meson
+hostmakedepends="libarchive-devel gtk+3-devel devhelp-devel webkit2gtk-devel
```
Comment (maxice8) [inline srcpkgs/quickDocs/template]: devel packages should be in makedepends=

File: srcpkgs/quickDocs/template (hunk 1 above)
Comment (TeddyDD) [reply srcpkgs/quickDocs/template]: wtf, why I forgot about this

File: srcpkgs/quickDocs/template [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'quicDocs'
+pkgname=quickDocs
+version=2.1.1
+revision=1
+build_style=meson
+hostmakedepends="meson vala pkg-config"
```
Comment (maxice8) [inline srcpkgs/quickDocs/template]: meson isn't needed on the hostmakedpends

**Pull request merged**